src/main/java/mate/academy/exception/DataProcessingException.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.exception;
+
+import java.sql.SQLException;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String warning, SQLException e) {
```

Comment:
```suggestion
    public DataProcessingException(String warning, Throwable e) {
```
let's have the opportunity to use this exception with a different cause as well

Reply:
Aha. Ok.